packages/worker/src/core.ts

```diff
@@ -8,22 +8,33 @@ import {
   normalizeTlsConfig,
 } from './server-options';
 import * as native from '@temporalio/core-bridge';
-import { newCore, coreShutdown, TelemetryOptions } from '@temporalio/core-bridge';
+import { newCore, coreShutdown, TelemetryOptions, corePollLogs } from '@temporalio/core-bridge';
+import { DefaultLogger, Logger } from "./logger";
+import { BehaviorSubject, interval } from 'rxjs';
+import { filter, takeUntil } from 'rxjs/operators';
 
 export interface CoreOptions {
   /** Options for communicating with the Temporal server */
   serverOptions?: ServerOptions;
   /** Telemetry options for traces/metrics/logging */
   telemetryOptions?: TelemetryOptions;
+  /**
+   * Custom logger for logging events from the SDK, by default we log everything to stderr
+   * at the INFO level.
+   */
+  logger?: Logger;
 }
 
 export interface CompiledCoreOptions extends CoreOptions {
   /** Options for communicating with the Temporal server */
   serverOptions: CompiledServerOptions;
+  logger: Logger;
 }
 
 function defaultTelemetryOptions(): TelemetryOptions {
-  return {};
+  return {
+    logForwardingLevel: 'OFF'
```

Comment:
Should this be OFF by default?
I thought that we want to forward the logs to unify the log formats we see in console (at least for development).

Reply:
Good point. For me I prefer having the extra info from the spans you get with the rust logging, but probably not for most people. Changed to `INFO` and I'll just override locally.